packages/expo-video/android/src/main/java/expo/modules/video/playbackService/ExpoVideoPlaybackService.kt

```diff
@@ -89,15 +110,52 @@ class ExpoVideoPlaybackService : MediaSessionService() {
   }
 
   override fun onUpdateNotification(session: MediaSession, startInForegroundRequired: Boolean) {
-    if (session.sessionExtras.getBoolean(SESSION_SHOW_NOTIFICATION, false)) {
-      createNotification(session)
-    } else {
-      (session.player as? ExoPlayer)?.let {
-        hidePlayerNotification(it)
+    appContext.mainQueue.launch {
+      if (startInForegroundRequired && session.wantsToShowNotification()) {
+        setMostRecentInteractionSession(session)
+      } else {
+        setMostRecentInteractionSession(findMostRecentInteractionSession())
       }
     }
   }
 
+  @MainThread
+  private fun setMostRecentInteractionSession(session: MediaSession?) {
+    if (isStateUnchanged(session)) {
+      return
+    }
```

Comment:
I did some further testing and this makes some of the artwork updates miss the notification. We can skip this check entirely to avoid such issues.
```suggestion
```